hack/verify-import-aliases.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Authors of Arktos.
```

Comment:
same here

Reply:
updated copyrights and hack/arktos_copyright_copied_k8s_files